### ISSUE TITLE
Change access level (public -> private) of context (DefaultJavaPrettyPrinter.java)

### ISSUE THREAD

**Issue by zielint0** [INRIA/spoon#2393]
Change access level from *public* to *private* in
 src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```
        /**
	 * The printing context.
	 *
	 * @deprecated since Spoon 7.1.0, please use {{@link #getContext()}}
	 */
	@Deprecated
	public PrintingContext context = new PrintingContext();
```
(to access a context, one uses *getContext()*)

The purpose of this issue is to remember of changing access of *context*.
Related PR: refactor: fix SonarQube - make field 'private', use accessor #2387

**Comment by surli** [comment]
I let the issue open for now, but FYI before releasing Spoon we always check the deprecated annotation to fix them if it's deprecated since more than one release.
So it will be necessarily done during this process.

**Comment by zielint0** [comment]
@surli 
Thank you for explanation. I was not sure about release process so created this issue.

**Comment by pvojtechovsky** [comment]
The DJPP#context is now private. Issue was solved by #2649
